README.md

```diff
@@ -34,6 +34,13 @@ You can pass `BUILD=debug` to the makefile for debug builds of both glad and gla
 sudo apt-get install libpulse0 libpulse-dev libglfw3 libglfw3-dev libxext6 libxext-dev libxcomposite-dev python make gcc 
 ```
 
+## Installation
+Some distributions have a package for `glava`. If your distribution is not listed please use the compilation instructions above.
+
+- Arch Linux [`glava` package](https://www.archlinux.org/packages/community/x86_64/glava/), or [`glava-git` AUR package](https://aur.archlinux.org/packages/glava-git/)
+- NixOS [package](https://github.com/NixOS/nixpkgs/blob/release-18.09/pkgs/applications/misc/glava/default.nix)
+- openSUSE [package](https://build.opensuse.org/package/show/X11:Utilities/glava)
```

Comment:
Your OpenSUSE link 404s

Reply:
I'm aware. But this will change once the [submit request](https://build.opensuse.org/request/show/653069) get accepted. Which usually takes only a few hours on a workday. But seems like since it was Friday it will be on Monday in case noone checks earlier. I still deemed it more reasonable to submit the final destination rather then my [home project](https://build.opensuse.org/package/show/home:jubalh/glava) to the readme.

Reply:
@coderobe what about this PR now?